runtime/tools/spec-builder/index.js

```diff
@@ -5,24 +5,30 @@ const curPath = process.cwd();
 
 let argv = require('yargs')
     .usage('Usage: $0 <command> [options]')
```

Comment:
1. I think that you have to remove `<command>` from this string because it shows as:

`Usage: spec-builder <command> [options]`

2. For some reason, the options for the output are not shown. Only the -f/--file, -h and --help are shown 
```
Usage: spec-builder <command> [options]

Options:
  --version   Show version number                                      [boolean]
  -f, --file  Load a file                                             [required]
  -h, --help  Show help                                                [boolean]

Examples:
  spec-builder -f customSpec.json -o        convert given file to raw and write
  customSpecRaw.json                        to output

copyright 2020 Limehchain LTD.
```

3. It will be cool to add a couple of empty lines for before the whole thing is printed if it's possible though.